// proto buffer syntax
syntax = "proto3";

// Server Methods Schema
service GreetService {
    rpc greet(GreetRequest) returns (GreetResponse);
    
}

// Request and Response Schema
message GreetRequest{
    string firstName = 1;
    string lastName = 2;
    int32 age = 3;
}

message  GreetResponse {
    string result = 1;
}